iCub_manual/conf_manual/iCubGazeboV2_5_visuomanip/wrappers/motorControl/alljoints-mc_remapper.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="alljoints-mc_remapper" type="controlboardremapper">
+    <paramlist name="networks">
+        <elem name="head_joints">       (  0  2 0 2 )</elem>
+        <elem name="left_arm_joints">   (  3  9 0 6 )</elem>
+        <elem name="right_arm_joints">  ( 10 16 0 6 )</elem>
+        <elem name="torso_joints">      ( 17 19 0 2 )</elem>
```

Comment:
Minor comment: if we use the `axesNames` parameter here we may be more robust in changes of order of the joints. Anyhow, that is fine.

Reply:
Thanks for the comment, I also disliked quite a bit messing with numbers every time. If there is an alternative I will try using it. I am not very familiar with these tags hence I just got a look from other files.

Reply:
See https://github.com/robotology/robots-configuration/blob/5acd4d69669a9a89de1417019d440a3c1f5a804c/ergoCubSN001/wrappers/motorControl/alljoints-mc_remapper.xml#L10 for an example of `controlboardremapper` + `axesNames`

Reply:
@traversaro I have changed it as suggested. Differently from the file you mentioned, there was no need to also have the parameter `<param name="joints"> <n_joints> </param>`, probably a leftover from a different configuration using `networks` intead of `axesNames`.

Reply:
You are right, that is not necessary. It was added there to make some CI of robots-configuration happy (see https://github.com/robotology/robots-configuration/pull/503), but we can remove them here.